### PR TITLE
OCPBUGS-9960: Add uninstall message annotation to csv

### PR DESCRIPTION
**Pull request by thejasn**

### CODE REVIEW

**Comment by swghosh** [comment]
/cc @xingxingxia 
for qe-approved
/cc @davemulford 
for px-approved

File: bundle/manifests/cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -213,6 +213,12 @@ metadata:
     containerImage: openshift.io/cert-manager-operator:latest
     createdAt: 2023-03-03T00:00:00
     olm.skipRange: <1.11.1
+    operator.openshift.io/uninstall-message: Operator cert-manager Operator for Red
+      Hat OpenShift will be removed from  cert-manager-operator namespace. If your
+      Operator configured any off-cluster resources, these will continue to run and
+      require manual cleanup. All operands created by the operator will need to be
+      manually cleaned up. Please refer to https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/cert-manager-operator-uninstall.html
+      for additional steps.
```
Comment (xingxingxia) [inline bundle/manifests/cert-manager-operator.clusterserviceversion.yaml:221]:
```suggestion
    operator.openshift.io/uninstall-message: The cert-manager Operator for Red
      Hat OpenShift will be removed from cert-manager-operator namespace. If your
      Operator configured any off-cluster resources, these will continue to run and
      require manual cleanup. All operands created by the operator will need to be
      manually cleaned up. Please refer to https://docs.openshift.com/container-platform/latest/security/cert_manager_operator/cert-manager-operator-uninstall.html
      for additional steps.
```

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -9,6 +9,12 @@ metadata:
     containerImage: ""
     createdAt: 2023-03-03T00:00:00
     olm.skipRange: <1.11.1
+    operator.openshift.io/uninstall-message: Operator cert-manager Operator for Red
```
Comment (xingxingxia) [inline config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:12]:
```suggestion
    operator.openshift.io/uninstall-message: The cert-manager Operator for Red
```

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -9,6 +9,12 @@ metadata:
...
+      Hat OpenShift will be removed from  cert-manager-operator namespace. If your
```
Comment (xingxingxia) [inline config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:13]:
```suggestion
      Hat OpenShift will be removed from cert-manager-operator namespace. If your
```

**Comment by xingxingxia** [comment]
Will do post-merge test.
/label qe-approved

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (thejasn) [reply config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml]: Good catch

**Comment by swghosh** [comment]
/label docs-approved
adding this label on my own as @xenolinux is OOO
(do doc changes would be required for this PR except for what is already in https://github.com/openshift/openshift-docs/pull/59057)

**Pull request merged**